src/goto-programs/goto_program.cpp

```diff
@@ -614,6 +630,8 @@ bool goto_programt::has_assertion() const
   return false;
 }
 
+/// Compute for each instruction the set of instructions of which it is a
+/// successor of.
```

Comment:
I think "of which" should be removed. But check with someone whose first language is English.